sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -16,3 +16,30 @@ export interface GetApplicationDataDto {
    */
   data: any;
 }
+
+export interface ApplicationAssessmentDTO {
+  /**
+   * Assessment data for an Application
+   */
+  weeks: number;
+  federal_assessment_need: number;
+  provincial_assessment_need: number;
+  total_federal_award: number;
+  total_provincial_award: number;
+}
+
+export class ApplicationDto {
+  /**
+   * Application dynamic data.
+   */
+  data: any;
+  /**
+   * Array of unique file names to be associated
+   * with this application.
+   */
+  associatedFiles: string[];
```

Comment:
Can you please elaborate on why data and associatedFiles are declared here?

Reply:
Removed it